apps/builder/server-express.js

```diff
@@ -0,0 +1,65 @@
+require("./env-check");
```

Comment:
- why is express server solving anything?

Reply:
🤷‍♂️  Remix just decided to throw server management on user.

Reply:
remix was never a server, it was app runtime, they had a server for development that afaik we have been using but now you are changing it for express server,so I wonder what happened